### ISSUE TITLE
Allow assignment of .vcml files to open in VCell in macs

### ISSUE THREAD

**Issue by ACowan0105** [virtualcell/vcell#187]
Need to check UCH spam server (digest.uchc.edu) for fix to reading VCML from web.

**Comment by jcschaff** [comment]
this may conflict with wanting to make VCell Client install as a non-administrator.

**Comment by ACowan0105** [comment]
@jcschaff yes but I also think it relates to the fact that we can't sign VCell to put as an app in the Apple app store.  This is a long standing problem with the greater powers that be at UConn and Ion put in a valiant effort but as I recall was never finally successful.  Perhaps we need to change the status as "blocked"?
